lib/chai/core/assertions.js

```diff
@@ -607,24 +900,46 @@ module.exports = function (chai, _) {
   Assertion.addProperty('Arguments', checkArguments);
 
   /**
-   * ### .equal(value)
+   * ### .equal(val[, msg])
+   *
+   * Asserts that the target is strictly (`===`) equal to the given `val`.
+   *
+   *     expect(1).to.equal(1);
+   *     expect('foo').to.equal('foo');
+   * 
+   * Add `.deep` earlier in the chain to use deep equality instead. See the
```

Comment:
I think deep equality is mentioned too many times throughout the documentation to go into detail about the algorithm. I prefer linking to the deep-eql project page instead where those kinds of details can go.